lib/pages/card_page_j.dart

```diff
@@ -23,22 +22,27 @@ class SimpleBusinessCardJ extends StatelessWidget {
           children: [
             CircleAvatar(
               backgroundImage: AssetImage(
-                "assets/images/avatar.jpeg",
+                Constants.user_image,
               ),
               backgroundColor: Colors.red,
               radius: Constants.radius,
             ),
-            Text(
-              'Juliana Dimeglio',
-              style: TextStyle(
-                fontSize: Constants.nameFontSize,
-                fontWeight: FontWeight.bold,
-                foreground: Paint()..shader = linearGradient,
+            Padding(
+              padding: const EdgeInsets.symmetric(
+                horizontal: Constants.padding,
+              ),
+              child: Text(
+                Constants.user_name,
```

Comment:
keep the same convention with the constants names, for the consistency of the code

Reply:
Fixed